extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3TaskLogsConfig.java

```diff
@@ -19,12 +19,14 @@
 
 package org.apache.druid.storage.s3;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import javax.validation.constraints.NotNull;
 
 /**
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Added @JsonIgnoreProperties annotation for backward compatibility as I've introduced disableAcl config.

Reply:
This shouldn't be required as we set:
```
configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```
in `DefaultObjectMapper`. Can you run a test and confirm this?

Reply:
Remove @JsonIgnoreProperties annotation as the required property is already set in the DefaultObjectMapper.

Reply:
Added a new test class with two test cases testing the functionality of the bucket owner having full control permissions on the S3 task log files in that bucket. The permissions will be set as if  disableAcl config is false. The test cases cover the cases with disableAcl true and false.